jena-arq/src/main/java/org/apache/jena/riot/Lang.java

```diff
@@ -153,6 +153,7 @@ public boolean equals(Object other) {
         Lang otherLang = (Lang)other ;
         return
             this.label == otherLang.label &&
+            this.altLabels.equals(otherLang.altLabels) &&
```

Comment:
Not sure it should even be testing alts. A lang corresponds to a MIME type registration which drives reader dispatch.

Has this been an issue for you?